src/stm32/stm32h7.c

```diff
@@ -210,6 +215,10 @@ clock_setup(void)
 static void
 usb_reboot_for_dfu_bootloader(void)
 {
+#ifdef STM32H723xx
```

Comment:
It's strange here. It's still in debugging. Any useful experience and suggestions are welcome